plugins/hu.bme.mit.massif.simulink.api/src/hu/bme/mit/massif/simulink/api/Exporter.java

```diff
@@ -493,19 +493,18 @@ private void exportBlocks(EList<Block> sameLevelBlocks) throws SimulinkApiExcept
             // //////////////////////////////////////////////////////////
 
             // Set block dialog/mask parameters
-            EList<Property> properties = block.getProperties();
-            for (Property property : properties) {
-                String propertyName = property.getName();
-                commandFactory.setParam().addParam(getFQN(block)).addParam(propertyName).addParam(property.getValue()).execute();
+            EList<Parameter> parameters = block.getParameters();
+            for (Parameter parameter : parameters) {
+                String propertyName = parameter.getName();
+                commandFactory.setParam().addParam(getFQN(block)).addParam(propertyName).addParam(parameter.getValue()).execute();
             }
 
         }
 
         // When there is at least one block on the same level, check that only blocks are there on the same level will be present in the exported model
         String containerFQN = getContainerFQN(sameLevelBlocks);
 
-        MatlabCommand findExistingBlocks = commandFactory.findSystem().addParam(containerFQN).addParam("IncludeCommented").addParam("on")
```

Comment:
You seem to be reverting the commented out block support.

Reply:
True, this should not be this way